File: hw/bonilla/U3/HW20Composite/Compositejs/model/Teller.js
const Employee = require('./Employee');

class Teller extends Employee {
  constructor(aName) {
    super();
    this.title = "Teller";
    if (aName) this.name = aName;
  }

  stateName() {
    super.stateName();
  }
}

module.exports = Teller;

File: hw/paillacho/U3/HW20Composite/Composite/src/main/java/ec/edu/espe/composite/model/Teller.java
package ec.edu.espe.composite.model;

/**
 *
 * @author Carlos Paillacho
 */
public class Teller extends Employee {

    /**
     * Default constructor sets the title.
     */
    public Teller() {
        title = "Teller";
    }

    /**
     * Constructs a Teller with a name.
     *
     * @param aName the teller's name
     */
    public Teller(String aName) {
        this();
        name = aName;
    }

    @Override
    public void stateName() {
        super.stateName();
    }
}
